server.js

```diff
@@ -0,0 +1,25 @@
+const express = require('express');
+const next = require('next');
+const dotenv = require('dotenv');
+
```

Comment:
try to use imports rather than require()

Reply:
removed